lib/services/AutoRenewService.ts

```diff
@@ -71,12 +71,12 @@ export class AutoRenewService implements ServiceInterface {
   }
 
   canStart() {
-    return (!!this.options.autoRenew || !!this.options.autoRemove);
+    return (!!this.options.autoRenew || !!this.options.autoRemove) && !this.started;
   }
 
   async start() {
+    await this.stop();
```

Comment:
I think we should be careful about calling `stop` in `start`. Since some of these services take a while to actually start (via Leader Election), I'm concerned calling `stop` in the middle of a election would cause some unpredictable behavior

Reply:
Removed `stop` in `start`